magic-modules/generate_terraform.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# This script takes in 'magic-modules', a git repo tracking the head of a PR against magic-modules.
+# It needs to output the same git repo, but with the code generation done, at 'mm-output'.
+
+# Setup GOPATH
+export GOPATH="${PWD}/go"
+WORKDIR="${PWD}"
+
+set -x
+set -e
+
+# Create $GOPATH structure - in order to successfully run Terraform codegen, we need to run
```

Comment:
This statement is not correct (at least it works for me on my environments.) Why do you believe it does not work?

Reply:
I'm really not sure what's going on there.  It's working now, though, changed the comment.